packages/block-editor/src/components/block-preview/index.js

```diff
@@ -29,10 +32,14 @@ export function BlockPreview( {
 	}
 	return (
 		<BlockEditorProvider value={ renderedBlocks } settings={ settings }>
-			<AutoHeightBlockPreview
-				viewportWidth={ viewportWidth }
-				__experimentalPadding={ __experimentalPadding }
-			/>
+			{ __experimentalLive ? (
+				<LiveBlockPreview onClick={ __experimentalOnClick } />
```

Comment:
Do we need the new live preview? It checked the [implementation](https://github.com/WordPress/gutenberg/blob/70a4af60c50926b696787a7f74c7eb58eb31e37a/packages/block-editor/src/components/block-preview/auto.js) of `AutoHeightBlockPreview`, and it's essentially also a `BlockList` wrapped in `Disabled`, plus some extra styling/responsiveness.

Reply:
(If this is only about the `onClick` prop, we might consider adding that to `AutoHeightBlockPreview` instead.)

Reply:
We don't need it to resize, and we can't memoize the block list.